### PR TITLE
PLATFORM-5301 | Jira error reporter: prioritize created issues

### PR DESCRIPTION
**Pull request by maaartyyynaa**

### CODE REVIEW

File: reporter/sources/ucp/errors.py [hunk 1]
```diff
@@ -110,9 +115,11 @@ def _get_report(self, entry):
             stack_trace=entry.get('stack_trace')
         ).strip()
 
+        priority = ERRORS_MAP[entry.get('event,type').replace("\"","")]
```
Comment (maaartyyynaa) [inline reporter/sources/ucp/errors.py:118]:
```suggestion
        priority = ERRORS_MAP[entry.get('event.type').replace("\"","")]
```

File: reporter/sources/ucp/errors.py [hunk 2]
```diff
@@ -110,9 +115,17 @@ def _get_report(self, entry):
             stack_trace=entry.get('stack_trace')
         ).strip()
 
+        priority = False
+        if entry.get('event') is not None and entry.get('event').get('type') is not None:
+            error_type = entry.get('event').get('type').replace("\"", "")
```
Comment (harnash) [inline reporter/sources/ucp/errors.py:120]:
```suggestion
            error_type = entry.get('event').get('type').replace('"', '')
```

**Review by harnash — APPROVED** [review]

**Comment by macbre** [comment]
Glad to see it's still used 🙂

**Comment by harnash** [comment]
> Glad to see it's still used 🙂

Resurrected but alive and kicking @macbre :)

**Pull request merged**